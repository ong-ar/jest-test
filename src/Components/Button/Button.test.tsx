import * as React from "react";
import * as renderer from "react-test-renderer";
import { shallow } from "enzyme";
import Button from "../Button";

it("swallow test", () => {
  const test = shallow(<Button title="dd" />);
  expect(test.text()).toEqual("d");
});

it("snapshot test", () => {
  const tree = renderer.create(<Button title="ddd" />).toJSON();
  expect(tree).toMatchSnapshot();
});
