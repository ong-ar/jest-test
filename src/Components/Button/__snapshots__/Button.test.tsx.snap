// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<header
  className="sc-bdVaJa bYVwNq"
>
  <h2
    className="sc-bwzfXH cKLoJe"
  >
    ddd
  </h2>
</header>
`;
